config/initializers/assets.rb

```diff
@@ -8,6 +8,6 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += ['about.css.sass', 'registration.css', 'report.css', '*.js', '*.coffee']
+Rails.application.config.assets.precompile += ['registration.css', 'report.css', '*.js', '*.coffee']
```

Comment:
Override the default setting provided by Hound: 
https://github.com/sul-dlss/sul-embed/blob/master/.rubocop.yml#L11-L13